### DISCUSSION TITLE
Recebimento PJ sem banco intermediario

### DISCUSSION THREAD

**Discussion by nativedone** [DyegoCosta/trabalhando-remoto#194] [General]
Fala galera!

Alguem aqui ja tah trampando na **Braintrust**?

Comei freelance lah 2 meses atras. O que me atraiu nesse marketplace foi:

-  eles nao descotam nenhuma taxa da gente (apenas clientes pagam taxa)
-  e eles tem clientes que pagam muito bem (50 a 150/h eh que geralmente vejo lah)

Meu unico problema ateh entao eh trazer o dinheiro pro BR. Acabei usando o payoneer porque o site da Braintrust nao tem opcao pra banco intermediario (veja as images abaixo). Pelo que li aqui nas outras issues tanto o BS2 quanto o remesa online usam bancos intermediarios.

No payonner nao tem banco intermediario mas em troca a coisa demora muito. Meu dinheiro estah lah jah fazem 2 dias como pendente e nada.

Esse marketplace eh bem novo e ainda nem lancaram e pra entrar vc precisa de um convite.
( spoiler eu tenho se alguem tiver interesse: https://app.usebraintrust.com/r/ad1/ )

**Mas, eu queria ver se alguem aqui jah esta usando Braintrust e achou uma forma rapida de receber como PJ.**

# Conta recebimento em dolar dentro dos EUA

![image](https://user-images.githubusercontent.com/20998754/101690229-fe662f80-3a4b-11eb-96e7-add246d41c2e.png)

# Conta recebimento em dolar fora dos EUA
![image](https://user-images.githubusercontent.com/20998754/101690452-10e06900-3a4c-11eb-9ec3-f14d0bdd3297.png)

**Comment by nativedone** [comment]
Update: acabei de receber essa info da minha gerente do RemessaOnline:

> Caso não tenha a opção do banco intermediário, pode cadastrar apenas o banco beneficiário ( Máxima) e direcionar a ordem direto ao banco Máxima junto ao seu IBAN. 

Alguem jah tentou usar o remessa online sem especificar o banco intermediario?

**Comment by enyamada** [comment]
Eu já, mas uma transferência pro Itaú. Na verdade eu não sabia, mas o Itaú tem um banco intermediário (se não me engano, uma agência do Wells Fargo) mas que só soube depois de fazer várias transferências sem nunca mencionar tal banco.

Se procurar no google, verá vários artigos mencionando não ser realmente necessário preencher o banco intermediário. Por ex, tem este post no quora: https://www.quora.com/What-will-happen-if-I-send-an-international-wire-transfer-without-specifying-the-intermediary-banking-information 

O ideal seria fazer uma transferência pequena de teste, mas se não for possível, há razões para acreditar que vai dar certo. Boa sorte.

**Comment by copercini** [comment]
Os únicos que conheço que são sem banco intermediário são o Banco Rendimento e o cotação (que tbm é do banco rendimento)

**Comment by nativedone** [comment]
@enyamada @copercini value pelas info. Vou ver se tento alguma dessas opcoes nos proximos pagamentos.

Update: Payoneer liberou meu pagamento. Demorou um total de 4 dias. Soh pra o dinheiro entrar na conta do Payoneer foi 1%. Pelas minhas simulacoes, pra trazer agora pro BR eh +2%. Entao, parece que vou ter um minino de 3% em taxas.

**Comment by manoelt** [comment]
@nativedone Não sei se ainda tem dúvida, mas eu fiz um SWIFT pela Remessa Online e não informei o banco intermediário, chegou no mesmo dia. 

Fiz o mesmo no Inter (por conflito de instruções do próprio banco) e foi uma dor de cabeça, tive que pedir recall do SWIFT através do pagador e o banco se negou até a ajudar.

**Comment by vrbrito** [comment]
Pessoal, alguém conhece algum banco que tenha spread de câmbio de menos de 2%? Também estou usando o payoneer e seria bem interessante se conseguisse transferir pra um banco direto que tivesse algo menor.

**Comment by endel** [comment]
Oi @vrbrito, no Remessas Online, dependendo do seu volume, você consegue negociar um cupom de desconto. 

Nessa transação, por exemplo, utilizei o cupom de desconto do blog "Mundo à Frente": https://www.youtube.com/watch?v=qWooGQGwvaY

<img width="222" alt="Screen Shot 2021-09-09 at 11 56 14" src="https://user-images.githubusercontent.com/130494/132710178-ef67df98-f41f-419f-9a24-497c39d92880.png">
